examples/supply-chain-app/package.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
Please divide codes depending on sample application to examples directory, not only package.json?

Reply:
> Could you mind if you divide codes depending on sample application to examples directory, not only package.json? This is the same comment as I said on contributors' meeting.

@takeutak It can be done, but it complicates the build to have it outside of the packages directory. Would you consider it separated based on the fact that the example packages literally have their folders prefixed with the word `example`?